calico-vpp-agent/policy/policy_server.go

```diff
@@ -58,13 +58,17 @@ const (
 	StateInSync
 )
 
+type NodeWatcherRestartError struct{}
```

Comment:
this error is returned in the policy server like any other felix message error and it results in restarting the policy server logic. Isn't that sufficient? or shall we kill the tomb to restart every thing?

Reply:
Yes, we should kill the tomb & ask wait for the scheduler to restart us. Otherwise we just restart felix's watcher (& regenerate events), but that will not update the node address use by other components